README.md

```diff
@@ -1,3 +1,52 @@
+
+In addition look for channels related to our working groups. As of early 2024 those are the Learning Path, Patterns, and ISPO group. Channels are named accordingly: #learning-path, #innersource-patterns, #ispo-working-group. In addition each of the groups has their own GitHub repository: [Learning Path](https://github.com/InnerSourceCommons/InnerSourceLearningPath), [Patterns](https://github.com/InnerSourceCommons/InnerSourcePatterns), [ISPO](https://github.com/InnerSourceCommons/ispo-working-group).
+
+If you want to help with outreach and marketing, feel free to join on Slack the #marketing channel.
+
+As a general rule of thumb: There's no expectation at all that new contributors start by sending us patches. Much like in open source you are invited to join our communication channels and spend some time watching over the shoulder. After a little while you will find yourself starting to answer incoming questions, provide insights to discussions initiated. Those are already very valuable contributions! Over time you may find yourself making modifications to our existing material - this can be as small as adding your organisation to the known instances section of a pattern, or as large as creating a learning path segment that you need for rolling out InnerSource in your organisation. Chances are high that you will find others in need for that material as well - and willing to collaborate on the material together with you.
+
+Donating time is the only way for you to become a member of the foundation and gain voting rights on new members and board members. One caveat though: All of the work shaping InnerSource happens withing the working groups. The foundation exists to enable those groups and provide some shelter from whatever legal risk there may be from working in the public.
+
+## Donate money
+
+Alternatively you can become one of our [sponsors](https://innersourcecommons.org/about/sponsors/) supporting us financially. This way you can show very visibly with your corporate affiliation that you support the InnerSource Commons.
+
+# Governance Structure
+
+All our initiatives are driven by volunteers in our working groups. As a user of our materials you may find room for improvement, like adding the name of your employer to the list of known instances in an existing pattern. When doing that you become a contributor to one of our working groups.
+
+Contributors who stick around over time are turned into Trusted Committers of the working group they are involved with. Typically Trusted Committers are also added to our GitHub organisation - both, to make interaction with their working group easier, as well as to publicly show their affiliation on their GitHub profile.
```

Comment:
How does the transition occur?  Is this automatic based on metrics or done subjectively by a human in the loop?

Reply:
At the InnerSource commons it is done by the humans in the working groups.

Ideally someone should suggest to turn the contributor into a new trusted committer. A vote occurs in private (to make sure no negative information on the person leaks into google searchable spaces), the person gets an invite and only if the accept the invite, that's published.

The risk that pure metrics based automatism here would be exploited just is too high.

Reply:
The formal process above to my knowledge isn't what's followed by all working groups, however those are often still tiny, so some flexibility should be fine.